### ISSUE TITLE
  Inference time by quantized model is longer than that by non-quantized model in Tensorflow

### ISSUE THREAD

**Issue by ZhangShuoAlreadyExists** [tensorflow/tensorflow#32904]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Ubuntu 1806
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:  ARMv8 AARCH64
- TensorFlow installed from (source or binary):  source
- TensorFlow version (use command below):  1.12.3
- Python version:  2.7.15
- Bazel version (if compiling from source):  0.15.0-dist   
- GCC/Compiler version (if compiling from source):  7.3
- CUDA/cuDNN version:  NA
- GPU model and memory: NA

You can collect some of this information using our environment capture
[script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)
You can also obtain the TensorFlow version with: 1. TF 1.0: `python -c "import
tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"` 2. TF 2.0: `python -c
"import tensorflow as tf; print(tf.version.GIT_VERSION, tf.version.VERSION)"`

**Describe the current behavior**
I want to measure Tensorflow inference time with different models on my 4-core ARMv8 CPU.
I downloaded quantized and float point models from https://www.tensorflow.org/lite/guide/hosted_models. Then run benchmark_model to measure inference time. 
Please see attached picture for my benchmarking result.
From the result, you can see the inference time of the quantized models is always a little bit longer than the float point model. 
I also did the same benchmark with TFLite, the inference time of the quantized models is much lower as I expect.

**Describe the expected behavior**
 I think the inference time of quantized models should be much lower than that of float point models.

**Code to reproduce the issue**
Provide a reproducible test case that is the bare minimum necessary to generate the problem.

The 
![tensorflow](https://user-images.githubusercontent.com/44886342/65829997-fc4c0100-e2dd-11e9-8bd5-c8a3d2b69e82.jpg)
commands as below:
For quantized models:
    echo "inception_v4"
    bazel-bin/tensorflow/tools/benchmark/benchmark_model --graph=model/classfication_int8/inception_v4/inception_v4.pb  --input_layer=input --input_layer_type=float --input_layer_shape=1,299,299,3 --output_layer=InceptionV4/Logits/Predictions  --show_run_order=false --num_threads=1 --show_flops

    echo "mobilenet_v1_1.0_224"
    bazel-bin/tensorflow/tools/benchmark/benchmark_model --graph=model/classfication_int8/mobilenet_v1_1.0_224/mobilenet_v1_1.0_224.pb --show_flops --input_layer=input --input_layer_type=float --input_layer_shape=1,224,224,3 --output_layer=MobilenetV1/Predictions/Reshape_1  --show_run_order=false --num_threads=1

    echo "mobilenet_v2_1.0_224"
    bazel-bin/tensorflow/tools/benchmark/benchmark_model --graph=model/classfication_int8/mobilenet_v2_1.0_224/mobilenet_v2_1.0_224.pb  --output_layer=output  --show_run_order=false --num_threads=1 --show_flops

For float point models:
    echo "inception_v4"
    bazel-bin/tensorflow/tools/benchmark/benchmark_model --graph=model/classfication-fp32/inception_v4/inception_v4.pb --show_flops --input_layer=input --input_layer_type=float --input_layer_shape=1,299,299,3 --output_layer=InceptionV4/Logits/Predictions  --show_run_order=false --num_threads=1

    echo "mobilenet_v1_1.0_224"
    bazel-bin/tensorflow/tools/benchmark/benchmark_model --graph=model/classfication-fp32/mobilenet_v1_1.0_224/mobilenet_v1_1.0_224.pb --show_flops --input_layer=input --input_layer_type=float --input_layer_shape=1,224,224,3 --output_layer=MobilenetV1/Predictions/Reshape_1  --show_run_order=false --num_threads=1

    echo "mobilenet_v2_1.0_224"
    bazel-bin/tensorflow/tools/benchmark/benchmark_model --graph=model/classfication-fp32/mobilenet_v2_1.0_224/mobilenet_v2_1.0_224.pb --show_flops --input_layer=input --input_layer_type=float  --output_layer=MobilenetV2/Predictions/Reshape_1  --show_run_order=false --num_threads=1

**Other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by suharshs** [comment]
Hi,

This is because the included models are not fully quantized for TensorFlow, but instead Quantization-aware trained model. This means they are float, but have FakeQuantWithMinMaxVars operations emulating the effect of quantization during training. The intention of these models is to emulate the effects of quantization in TensorFlow graphs so that they can be easily converted to TensorFlow Lite models.

Thanks!
